07.high-fidelity-commerce/01.basics/01.hfc/02.send-money/docs.md

```diff
@@ -12,10 +12,9 @@ This page details how you can transfer HFC from your wallet to another user.
 
 ## Introduction
 
-Once you are done with setting up your wallet and receiving your HFC, learn how to send HFC to another user. 
+Once you have logged into High Fidelity and [received your HFC](http://localhost:8000/high-fidelity-commerce/basics/hfc/get-hfc), learn how to send HFC to another user. 
```

Comment:
Will this link get updated when this PR is pushed live?

Reply:
Fixed